WooCommerce/Classes/ViewRelated/Orders/Order Details/Receipts/ReceiptEmail/ReceiptEmailViewModel.swift

```diff
@@ -1,24 +1,36 @@
 import Foundation
 import class WordPressShared.EmailFormatValidator
 import Yosemite
+import WooFoundation
+import Combine
 
 final class ReceiptEmailViewModel: ObservableObject {
     @Published var email: String = ""
     @Published var state: PrimaryLoadingButtonStyle.State = .idle
+    @Published private(set) var dismiss: Bool = false
 
-    private let order: Order
+    private var order: Order
     private let stores: StoresManager
+    private let analytics: Analytics
+    private let countryCode: CountryCode
+    private let cardReaderModel: String?
+    var onDismiss: (Order?) -> Void
```

Comment:
nit: this could be let?
```suggestion
    let onDismiss: (Order?) -> Void
```